src/main/java/com/savvato/basemobileapp/controllers/SMSChallengeCodeAPIController.java

```diff
@@ -21,32 +24,42 @@ public class SMSChallengeCodeAPIController {
     SMSChallengeCodeService smsccs;
 
     @RequestMapping(value = { "/api/public/sendSMSChallengeCodeToPhoneNumber" }, method=RequestMethod.POST)
-    public String sendSMSChallengeCode(@RequestBody @Valid SMSChallengeRequest req) {
+    public ResponseEntity<GenericResponseDTO> sendSMSChallengeCode(@RequestBody @Valid SMSChallengeRequest req) {
         String phoneNumber = req.phoneNumber;  // assume the number we're getting is 10 digits, without the country code
 
         if (!phoneNumber.startsWith("0"))
             phoneNumber = "1" + phoneNumber;
 
-        String rtn = smsccs.sendSMSChallengeCodeToPhoneNumber(phoneNumber);
-        log.debug("Sent challenge code to " + phoneNumber + ". " + rtn);
-        return rtn;
+        GenericResponseDTO genericResponseDTO = GenericResponseDTO
+                .builder()
+                .responseMessage(smsccs.sendSMSChallengeCodeToPhoneNumber(phoneNumber))
```

Comment:
Here as well, use a variable.

Reply:
Or alternatively, you could inline the other places where you did use a variable. Not as helpful for the next developer, but not a bad choice either. Consistency is the key.